File: src/main/java/com/zunza/buythedip/watchlist/repository/WatchlistItemRepository.java
package com.zunza.buythedip.watchlist.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.zunza.buythedip.watchlist.entity.WatchlistItem;

@Repository
public interface WatchlistItemRepository extends JpaRepository<WatchlistItem, Long> {
}

File: src/main/kotlin/com/zunza/buythedip/watchlist/repository/WatchlistItemRepository.kt
package com.zunza.buythedip.watchlist.repository

import com.zunza.buythedip.watchlist.entity.WatchlistItem
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface WatchlistItemRepository : JpaRepository<WatchlistItem, Long> {
}
